pinot-common/src/main/java/com/linkedin/pinot/common/response/ServerInstance.java

```diff
@@ -204,4 +206,16 @@ public static ServerInstance forHostPort(String name, int port) {
       return newInstance;
     }
   }
+
+  public static ServerInstance forInstanceName(String instanceName) {
```

Comment:
Check if the optimization made in https://github.com/linkedin/pinot/pull/1487 is required here as well.